### ISSUE TITLE
On ubuntu Xenial, osdlyrics does not detect anything from multiple players

### ISSUE THREAD

**Issue by nicman23** [osdlyrics/osdlyrics#14]
Have tried VLC, mpv, and banshee. Nothing seams to work.

I have installed the dependencies that are listed in the aur PKGBUILD and compiled with a standard debuild after adding --with autoreconf and --with python2 to the debian/rules.

I also tried with manually with ./autogen.sh ./configure --prefix=/usr and make -j5 ; make install

Nothing seams to work, there is no output in stdout / stderr

Installing / compiling the old 0.4.3 works, but the online sources do not.

**Comment by PedroHLC** [comment]
"master" branch only implements MPRIS 2-, MPD and HTTP by now.
https://github.com/osdlyrics/osdlyrics/tree/master/players

**Comment by lenoch** [comment]
To be honest, I have been using osdlyrics with MPRIS1 only (in MOC). However, I confirm VLC did not work for me until I ran a debugger and fixed MPRIS2 :-)

Before I post a patch, the code I had to fix was actually in python/player_proxy.py
I believe that the usage of names such as __connected is quite dangerous due to name mangling. So I carefully replaced all double underscores (not at the end, though: __init__) with single underscores. I'll post a patch soon.
